lib/utils/add-token-to-user-doc.js

```diff
@@ -15,6 +15,9 @@ function addTokenToUserDoc (db, account, token) {
     var id = token.id || uuid.v4()
     delete token.id
 
+    var now = new Date(Date.now())
```

Comment:
you can leave out the `Date.now()`, `new Date()` should do the same thing.
You can also shorten the two lines to ` token.createdAt = new Date().toISOString()` if you don’t use the `now` variable anywhere else